### DISCUSSION TITLE
Follower : add few new commands ?

### DISCUSSION THREAD

**Discussion by tapakahxp** [blizzhackers/kolbot#378] [Ideas]
Is it possible to add few new commands for follower bot ?

"Chug": Get and drink some Thawing and Antidote potions from nearest merchant.
"ShareGold": Divide equal amount of gold among all members. Since some have 200k, some have 1k
"Talk Cain": After killing councils in act 3.

**Comment by theBGuy** [comment]
Can you just join the discord? Half of these issues you've been opening up are just spam that could of been answered easily in chat. Short answer is yes, besides the gold. It could be done but it'd be annoying to do as it would require communication between all of them.

**Comment by theBGuy** [comment]
talking is fixed now I had broken it in a previous commit, also added the chug command

**Comment by theBGuy** [comment]
I'm wondering about a taxi command but trying to conceptualize its usage. I can do 
`taxi <areaid>` easy enough where a character(s) would journey to that area then make a portal but not sure that's all that useful as you would need to know the area id's and it seems limited. Or would it be better to just do
`taxi <some specific place>` i.e `taxi throne` or `taxi andy`.

**Comment by tapakahxp** [comment]
> Can you just join the discord?

I dont even know what is it. However, i supose all this topics is for all members, not just me.

> added the chug command

That's great.
I hope bots won't stuck at Aldor on Act 3, as they usually do when getting quest 1 potions (+20 hp)

> I'm wondering about a taxi command

Isn't it second rusher bot ?
Or you mean, all followers should get waypoints to the area you typed ?
Word "taxi throne" would be much better, but in case of number, script can assume AreaId

**Reply by theBGuy** [reply]
1) quick google search would tell you what discord is, or clicking on the link in the readMe
2) If you are running with multiple bots, never send them all over to talk to alkor at once. That's just bad on your part, can't fix a small building and a small pathway. 
3) No it wouldn't be rusher addition, the idea behind the taxi command would be manually playing a non-tele character and being able to command a follower who has tele to go to certain places
